Services/UserCourseStatusHistorizing/classes/class.ilUserCourseStatusHistorizingHelper.php

```diff
@@ -220,6 +220,14 @@ public function setIndividualStartAndEnd($user_id, $course_id, &$payload) {
 			$payload["begin_date"] = date("Y-m-d");
 			return;
 		}
+
+		if (ilUserCourseStatusHistorizing::caseExists($case_id)
+			&& $payload["booking_status"] == "status_cancelled_without_costs"
```

Comment:
Gibt es den Status? Hab in der Live Datenbank mal einen DISTINCT drauf geworfen und das kam dabei nicht rum.